File: Python/sexyPrime.py
def sexy_prime(x, y):
    sexy = False
    if isPrime(x) == False or isPrime(y) == False:
        return False
    elif isPrime(x) == True and isPrime(y) == True:
        if y - x == 6 or x + 6 == y or y+6 == x:
            sexy = True
        elif y - x != 6:
            sexy = False
    return sexy


def isPrime(n):
    if n==2 or n==3: return True
    if n%2==0 or n<2: return False
    for i in range(3, int(n**0.5)+1, 2):   # only odd numbers
        if n%i==0:
            return False    

    return True

File: JavaScript/sexyPrime.js
function sexy_prime(x, y){
    if(x < 2 || y < 2) return false
  
    function prime(n){
      for(var i=2;i<n;i++){
        if(n % i == 0) return false
      }
      return true
    }
  
    var arr = []
    for(var i=x;i<y;i++){
        if(prime(i)) arr.push(i)
    }
    
    if(!prime(x)||!prime(y)) return false
    else if(x + 6 == y || y + 6 == x){
      return true
    }
    return false
  }